### PR TITLE
use CircleCI to update gh-pages with latest downloadable files in YAML, JSON, CSV

### PR DESCRIPTION
**Pull request by JoshData**
For #480: Created a circle.yml file to use CircleCI to automatically push updated YAML, JSON, and CSV files to gh-pages. And added download links to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,28 @@
 congress-legislators
 ====================
 
-Members of the United States Congress (1789-Present) and congressional committees (1973-Present) in YAML.
+Members of the United States Congress (1789-Present), congressional committees (1973-Present), committee membership (current only), and presidents and vice presidents of the United States in YAML, JSON, and CSV format.
 
-### Using the data
-
-This repository contains data about legislators...:
+[![Build Status](https://secure.travis-ci.org/unitedstates/congress-legislators.png)](http://travis-ci.org/unitedstates/congress-legislators)
```
Comment (konklone) [inline README.md]: I guess we should probably just start using Circle for all build testing too (in another PR, just noting).

File: circle.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# CircleCI configuration file.
+machine:
+  python:
+    version: 3.5.2
```
Comment (konklone) [inline circle.yml]: Just noting for future reference that Python 3.6.1 is out (and available on Circle).

File: scripts/update_gh_pages.sh [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+# Generate CSV and JSON.
+(cd scripts/; python alternate_bulk_formats.py;)
+
+# Commit the YAML, CSV, and JSON.
+# (Don't commit the other scripts files we checked out from
+# the source branch, which git has unhelpfully put in the
+# index.)
+export GIT_AUTHOR_NAME="the unitedstates project (CircleCI)"
+export GIT_AUTHOR_EMAIL=circleci@theunitedstates.io
```
Comment (konklone) [inline scripts/update_gh_pages.sh]: Noting that maybe we should formalize this email somehow in the future.

**Comment by konklone** [comment]
There are some merge conflicts in the data, but I'm a :+1: on the PR overall, feel free to merge after resolving the conflicts.

**Comment by JoshData** [comment]
Thanks for looking it over. I'm going to give it another look to make sure it's really working... it's been hard to get this right... then ill merge.

On June 27, 2017 11:03:40 AM EDT, Eric Mill <notifications@github.com> wrote:
>There are some merge conflicts in the data, but I'm a :+1: on the PR
>overall, feel free to merge after resolving the conflicts.
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/unitedstates/congress-legislators/pull/482#issuecomment-311387023

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by JoshData** [comment]
Ok this is merged.

The repo README now has a table with links to YAML, JSON, and CSV files, which are hosted on our gh-pages branch. Every push to master triggers a kind of complicated script run by CircleCI to generate the JSON and CSV and commit & push to gh-pages.

We no longer have to keep the JSON files updated ourselves - the JSON files have been deleted from master.

**Pull request merged**